### PR TITLE
Create interfaces/classes and services

### PR DESCRIPTION
**Pull request by miguelhenaoa**
### Summary
- Add Gender, Species and Status enums
- Add CharacterResponseData, Character, FilterParams and Info interfaces
- Add API_URL in environment files
- Add API service

### Trello Card
https://trello.com/c/llt4tKzB

### CODE REVIEW

File: src/app/interfaces/character.interface.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+interface Origin {
+  name: string;
+  url: string;
+}
+
+interface Location {
+  name: string;
+  url: string;
+}
```
Comment (andres-acc) [inline src/app/interfaces/character.interface.ts:13]: This interfaces could be a single common interface called 'Place'.

File: src/app/interfaces/info.interface.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export interface Info {
+  count: number;
+  pages: number;
+  next: string | null;
+  prev: string | null;
+}
```
Comment (andres-acc) [inline src/app/interfaces/info.interface.ts:6]: This name could be better. ResponseProperties or something like that.

**Review by andres-acc — APPROVED** [review]
Looks good, Just a couple of suggestion in the naming.

File: src/app/app.module.ts [hunk 3]
```diff
@@ -1,18 +1,14 @@
...
 @NgModule({
-  declarations: [
-    AppComponent
-  ],
-  imports: [
-    BrowserModule,
-    AppRoutingModule
-  ],
+  declarations: [AppComponent],
+  imports: [BrowserModule, AppRoutingModule, HttpClientModule],
```
Comment (pedrortizm18) [inline src/app/app.module.ts:10]:
```suggestion
  imports: [
    BrowserModule, 
    AppRoutingModule, 
    HttpClientModule
  ],
```

File: src/app/services/api.service.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
```
Comment (pedrortizm18) [inline src/app/services/api.service.ts:3]:
```suggestion
import { Observable } from 'rxjs';
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';
```

**Review by pedrortizm18 — APPROVED** [review]

**Pull request merged**